python/py-pathvalidate/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-pathvalidate
-version             2.5.2
+version             3.0.0
```

Comment:
I didn't check, but I guess this will no longer be API compatible. Do we have dependents?

Reply:
@raimue only one port `qobuz-dl` and it works :)

Reply:
BTW, based on https://github.com/thombashi/pathvalidate/releases/tag/v3.0.0 changes aren't that huge and should be safe